recipe/meta.yaml

```diff
@@ -40,7 +43,7 @@ test:
 about:
   home: https://github.com/MDAnalysis/mdaencore
   summary: Ensemble overlap comparison software for molecular data.
-  license: GPL-2.0-or-later
+  license: GPL-3.0-or-later
   license_file: LICENSE
```

Comment:
If the source code includes a GPLv2+ LICENSE file then we need to include the GPLv3+ file here separately.

Reply:
Yeah I guess we should - I've never really fully understood the packaging requirements for GPL when you upgrade, but it would be best

Reply:
thanks for adding @ianmkenney